frontend/src/components/events/molecules/event_card.css

```diff
@@ -18,14 +18,39 @@
   opacity: 0;
   transition: 0.5s ease;
   background-color: gray;
-  overflow: auto;
   border-radius: 5px;
 }
 
 .eventCard:hover .overlay {
-  opacity: 0.9;
+  opacity: 0.95;
 }
 
 .eventTextBoxes {
   height: 110px;
 }
+
+.learnMore {
+  position: absolute;
+  width: max-content;
+  bottom: 20px;
+  right: 97.5px;
```

Comment:
we can do this a bit easier by creating a parent div

so `<div class="learnMore"><btn></btn></div>`

then do this css like so 

```
.learnMore {
    position: absolute;
    bottom: 20px;
    width: 100%;

    btn {
          position: relative;
          margin: auto;
         display: flex;
         background: #519b56;
         border-color: #eeeaea;
         color: #eeeaea;
         font-weight: bold;
    }
}
```